go/vt/vttest/environment.go

```diff
@@ -235,21 +245,31 @@ func tmpdir(dataroot string) (dir string, err error) {
 // randomPort gets a random port that is available for a TCP connection.
 // After we generate a random port, we try to establish tcp connections on it and the next 5 values.
```

Comment:
Does that mean we don't end up using five ports here? So, we use the `randomPort` that we generated, but always leave a gap of five more ports after it?